Amazon.QLDB.Driver/transaction/AsyncTransaction.cs

```diff
@@ -25,13 +25,11 @@ namespace Amazon.QLDB.Driver
     using Microsoft.Extensions.Logging;
 
     /// <summary>
-    /// Implementation of a QLDB transaction which also tracks child Results for the purposes of managing their lifecycle.
-    /// Any unexpected errors that occur when calling methods in this class should not be retried, as the state of the
-    /// transaction is now ambiguous. When an OCC conflict occurs, the transaction is closed.
-    ///
-    /// Child Result objects will be closed when the transaction is aborted or committed.
```

Comment:
Are child result objects no longer closed?

Reply:
Result objects do not have an explicit state in .NET like they do in Java, due to the absence of Read-ahead.